modules/programs/wezterm.nix

```diff
@@ -79,6 +79,7 @@ in {
       '';
     };
 
+    enableBashIntegration = mkEnableOption "WezTerm's Bash integration.";
```

Comment:
It seems like the provided script can also handle Zsh, could you add that?

Also, is there any reason not to have this default to `true`?

Reply:
> It seems like the provided script can also handle Zsh, could you add that?

Sure.

> Also, is there any reason not to have this default to true?

None that I know of. Enabled it by default.